tutorials/R/download-ais-data/download-NEON-AIS-data.Rmd

```diff
@@ -0,0 +1,307 @@
+choice because it ensures you're always working with the most up to date data, 
+and it ends with ready-to-use tables in R. However, if you use it in
+a workflow you run repeatedly, keep in mind it will re-download the 
+data every time.
+
+Before we get the NEON data, we need to install (if not already done) and load 
+the neonUtilities R package. 
+
+```{r set-up-env, eval=F}
+# Install neonUtilities package if you have not yet.
+install.packages("neonUtilities")
+install.packages("ggplot2")
+
+```
+
+```{r load-packages}
+# Set global option to NOT convert all character variables to factors
+options(stringsAsFactors=F)
+
+# Load required packages
+library(neonUtilities)
+library(ggplot2)
+
+```
+
+The inputs to `loadByProduct()` control which data to download and how 
+to manage the processing. The following are frequently used inputs: 
+
+* `dpID`: the data product ID, e.g. DP1.20288.001
+* `site`: defaults to "all", meaning all sites with available data; 
+can be a vector of 4-letter NEON site codes, e.g. 
+`c("MART","TOMB","BARC")`.
+* `startdate` and `enddate`: defaults to NA, meaning all dates 
+with available data; or a date in the form YYYY-MM, e.g. 
+2017-06. Since NEON data are provided in month packages, finer 
+scale querying is not available. Both start and end date are 
+inclusive.
+* `package`: either basic or expanded data package. Expanded data 
+packages generally include additional information about data 
+quality, such as chemical standards and quality flags. Not every 
+data product has an expanded package; if the expanded package is 
+requested but there isn't one, the basic package will be 
+downloaded.
+* `avg`: defaults to "all", to download all data; or the 
+number of minutes in the averaging interval. See example below; 
+only applicable to IS data.
+* `savepath`: the file path you want to download to; defaults to the 
+working directory.
+* `check.size`: T or F; should the function pause before downloading 
+data and warn you about the size of your download? Defaults to T; if 
+you are using this function within a script or batch process you 
+will want to set this to F.
+* `token`: this allows you to input your NEON API token to obtain faster downloads. 
+Learn more about NEON API tokens in the <a href="https//:www.neonscience.org/neon-api-tokens-tutorial" target="_blank">**Using an API Token when Accessing NEON Data with neonUtilities** tutorial</a>. 
+
+There are additional inputs you can learn about in the 
+<a href="https//:www.neonscience.org/neonDataStackR" target="_blank">**Use the neonUtilities R Package to Access NEON Data** tutorial</a>. 
+
+The `dpID` is the data product identifier of the data you want to 
+download. The DPID can be found on the 
+<a href="http://data.neonscience.org/data-products/explore" target="_blank">
+Explore Data Products page</a>.
+
+It will be in the form DP#.#####.###. For this tutorial, we'll use three data
+product: 
+
```

Comment:
Should the other two data products be specified here?

Reply:
Hold over from earlier version - removed reference to 3.